2.11.0/jiva-operator-RC1.yaml

```diff
@@ -0,0 +1,4045 @@
+  - secrets
+  verbs:
+  - '*'
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  - daemonsets
+  - replicasets
+  - statefulsets
+  verbs:
+  - '*'
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
+  verbs:
+  - get
+  - create
+- apiGroups:
+  - apps
+  resourceNames:
+  - jiva-operator
+  resources:
+  - deployments/finalizers
+  verbs:
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - apps
+  resources:
+  - replicasets
+  verbs:
+  - get
+- apiGroups:
+  - policy
+  resources:
+  - poddisruptionbudgets
+  verbs:
+  - '*'
+- apiGroups:
+  - openebs.io
+  resources:
+  - '*'
+  verbs:
+  - '*'
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: jiva-operator
+  namespace: openebs
+subjects:
+- kind: ServiceAccount
+  name: jiva-operator
+  namespace: openebs
+roleRef:
+  kind: ClusterRole
+  name: jiva-operator
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jiva-operator
+  namespace: openebs
+  labels:
+    openebs.io/component-name: jiva-operator
+    openebs.io/version: 2.11.0-RC1
+    name: jiva-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: jiva-operator
+  template:
+    metadata:
+      labels:
+        name: jiva-operator
+        openebs.io/component-name: jiva-operator
+        openebs.io/version: 2.11.0-RC1
+    spec:
+      serviceAccountName: jiva-operator
+      containers:
+        - name: jiva-operator
+          # Replace this with the built image name
+          image: openebs/jiva-operator:2.11.0-RC1
+          command:
+          - jiva-operator
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: OPENEBS_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: OPENEBS_SERVICEACCOUNT_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.serviceAccountName
+            - name: OPERATOR_NAME
+              value: "jiva-operator"
+            - name: OPENEBS_IO_JIVA_CONTROLLER_IMAGE
+              value: "openebs/jiva:2.11.0-RC1"
+            - name: OPENEBS_IO_JIVA_REPLICA_IMAGE
+              value: "openebs/jiva:2.11.0-RC1"
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8282
+            initialDelaySeconds: 15
+            periodSeconds: 20
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: 8282
+            initialDelaySeconds: 5
+            periodSeconds: 10
+          resources:
+            limits:
+              cpu: 100m
+              memory: 30Mi
+            requests:
+              cpu: 100m
+              memory: 20Mi
```

Comment:
Can we remove this for now? The pod sometimes crashes due to memory shortage. I will add it back once I test out correct limits

Reply:
done